tests/core/mempool/test_mempool_manager.py

```diff
@@ -53,7 +53,7 @@ async def get_coin_record_for_test_coins(coin_id: bytes32) -> Optional[CoinRecor
     return test_coin_records.get(coin_id)
 
 
-def create_test_block_record(*, height: uint32 = TEST_HEIGHT) -> BlockRecord:
+def create_test_block_record(*, height: uint32 = TEST_HEIGHT, timestamp: uint64 = TEST_TIMESTAMP) -> BlockRecord:
```

Comment:
it doesn't look like you're using this variable anywhere.
if you add `, *` before the `timestamp` parameter, you'll prevent it being specified as a positional argument, but has to be named if specified

Reply:
Done.